patch/timer/dev/16bit-tiny861-tc0.yaml

```diff
@@ -1,13 +1,6 @@
-TCCR?A:
-  _modify:
-    COM??:
-      _write_constraint: enum
-  COM??:
-    _replace_enum:
-        DISCONNECTED:  [0, "Normal port operation, OCix disconnected"]
-        MATCH_TOGGLE:  [1, "Toggle OCix on Compare Match (Might depend on WGM)"]
-        MATCH_CLEAR:   [2, "Clear OCix on Compare Match (If PWM is enabled, OCix is set at BOTTOM)"]
-        MATCH_SET:     [3, "Set OCix on Compare Match (If PWM is enabled, OCix is cleared at BOTTOM)"]
+_modify:
+  TIFR?:
+    access: read-write
```

Comment:
Just for my understanding: The tiny861 TC0 doesn't have any `TCCR#A[COM##]` fields and thus needs a separate patch.  Is that correct?

Reply:
The t861 16-bit timer is a little weird compared to some of the other 16-bit timers, and I think this is because it can be split into 2 8-bit timers (with some caveats). I haven't come across this in any other chips, so should I maybe have put it into `patch/timer/attiny861.yaml` instead of a separate timer file, kinda like how it's done it `patch/timer/atmega328pb.yaml`? If so, I think it's something that could be done in a separate commit, since the tiny88 and tiny85 also have separate timer files.

Either way, I think the answer to your question is yes, `TCCR0A` does not contain `COM##` fields, or any other multi-bit fields that are enumerated.